web/ci.sh

```diff
@@ -24,15 +24,22 @@ cd "$THIS_SCRIPT_PATH"
 
 builder_describe "Defines and implements the CI build steps for Keyman Engine for Web (KMW)." \
   "build" \
-  "test             Runs all unit tests."  \
-  "post-test        Runs post-test cleanup.  Should be run even if a prior step fails." \
-  "validate-size    Runs the build-size comparison check" \
-  "--debug          Runs this script in local-development mode; reports and tests will be locally logged"
+  "test                 Runs all unit tests."  \
+  "post-test            Runs post-test cleanup.  Should be run even if a prior step fails." \
+  "validate-size        Runs the build-size comparison check" \
+  "publish-s.keyman     Prepares an s.keyman.com PR (intended for release builds)" \
```

Comment:
I wonder if @mcdurdin has plans to have a "publish" action for builder scripts. Possibly then we'd have targets `:s.keyman` or `:downloads`

Reply:
The issue is that adding targets tends to propagate them to all the other actions, too.  I don't think it's as big of a problem as it used to be, though, so maybe I could swap to that.

Reply:
I was thinking of making 'publish' a separate script, because it should only ever be run on the build agents.

Reply:
This is all about a `ci.sh`.  It _is_ the separate script.

Reply:
I would like to have `:s.keyman.com` and `:downloads.keyman.com` as targets, and use `publish` as the action.